modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/describer/MuleExtensionAnnotationParser.java

```diff
@@ -127,6 +129,31 @@ public static Extension getExtension(Class<?> extensionType)
         return parsedParameters;
     }
 
+    static <T extends Annotation> List<T> parseRepeatableAnnotation(Class<?> extensionType, Class<T> annotation,
+                                                                    Function<Annotation, T[]> containerConsumer)
+    {
+        List<T> annotationDeclarations = Collections.emptyList();
+
+        Repeatable repeatableContainer = annotation.getAnnotation(Repeatable.class);
+        if (repeatableContainer != null)
+        {
+            //TODO FIXME look parent objects
```

Comment:
This has to be updated once https://github.com/mulesoft/mule/pull/2554 is merged, in order to rehuse the annotation lookup when inheritance is used